ql/test/experimental/frameworks/Gin/Gin.go

```diff
@@ -0,0 +1,251 @@
+package main
+
+// depstubber -vendor github.com/gin-gonic/gin Context
+// depstubber -vendor github.com/gin-gonic/gin/binding "" YAML
```

Comment:
```suggestion
//go:generate depstubber -vendor github.com/gin-gonic/gin Context
//go:generate depstubber -vendor github.com/gin-gonic/gin/binding "" YAML
```

Sorry, I got this syntax wrong somehow....